src/pool/srv_pool.c

```diff
@@ -1692,7 +1692,7 @@ ds_pool_create_handler(crt_rpc_t *rpc)
 
 static int
 pool_connect_bcast(crt_context_t ctx, struct pool_svc *svc,
-		   const uuid_t pool_hdl, uint64_t capas,
+		   const uuid_t pool_hdl, uint64_t capas, d_iov_t creds,
```

Comment:
Doesn't look like we're actually using it yet in the code below. This function is where tci_cred should be populated, right?

Reply:
That is correct. THat must have gotten lost in one of my rebases.